hw1/task1/stats/trip_stat.csv

```diff
@@ -0,0 +1,4 @@
+month,passenger_count,trip_distance
+4,1,2.17125
+4,3,6.39
+4,4,1.3
```

Comment:
I've got following results during my run:
```month,passenger_count,trip_distance
4,1.0,2.17125
4,3.0,6.39
4,4.0,1.3```
can you fix the format to not have decimal part?